.gitignore

```diff
@@ -30,3 +30,6 @@ bin/generated-sources/
 
 applicationinsights-agent-*.jar
 *.log
+/src/main/resources/application-perf.yaml
+/src/main/resources/application-dets.yaml
+/src/main/resources/application-demo.yaml
```

Comment:
What are these for?

Reply:
these are files i have locally that i don't want to ever accidentally commit, so i am telling git to ignore.

Reply:
It would be better to add these to your `~/.gitignore` file, so they are only ignored for you.

Reply:
removed